### ISSUE TITLE
UIBarButtonItem 

### ISSUE THREAD

**Issue by HackShitUp** [gemr/Sheriff#14]
Can you set the badge icon on a UIBarButtonItem?

**Comment by mamaral** [comment]
No, because UIBarButtonItem inherits from the abstract UIBarItem class, which is simply an NSObject, and as such has no frame or other UIView attributes. Perhaps we should update the tagline to: *Badgify anything. (as long as it's a UIView subclass)*...

:)

**Comment by nick-iCars** [comment]
sad face
